pkg/nats/proxy/constants.go

```diff
@@ -33,3 +36,11 @@ func callbackPublishSubject(nodeID string, method string) string {
 func callbackSubscribeSubject(nodeID string) string {
 	return fmt.Sprintf("%s.%s.>", CallbackSubjectPrefix, nodeID)
 }
+
+func registrationPublishSubject(method string) string {
+	return fmt.Sprintf("%s.%s", RegistrationSubjectPrefix, method)
```

Comment:
I suggest to include the compute node ID's in the subject to allow doing ACL in the future where a compute node can only publish and subscribe to subjects in the pattern `node.registeration.{node_id}.>`